### DISCUSSION TITLE
Eastern NC node network.

### DISCUSSION THREAD

**Discussion by DUCKYxACTUAL** [meshtastic/Discussions#100] [North America]
There was a guy on discourse talking about starting a node network in Wilmington NC, went by user name drz. You on here bud? Im looking to try and get some nodes up in the onslow area. Maybe we could help each other.

**Comment by MRK2340** [comment]
Hello, new here and to Meshtastic but if I can help out I would love to help start a network in the area. I am in burgaw NC. I just got four heltec radio so looking to start.
